src/mame/subsino/subsino.cpp

```diff
@@ -3406,10 +3406,10 @@ ROM_START( newhunterc )
 	ROM_LOAD( "n82s129an.u3",  0x00200, 0x00100, CRC(a1c0d069) SHA1(794df68451525901ebd5895feb26fcda2c517c3f) )
 
 	ROM_REGION( 0x00045c, "plds", 0 )
-	ROM_LOAD( "gal16v8b.u41",  0x00000, 0x00117, NO_DUMP )
-	ROM_LOAD( "gal16v8b.s63",  0x00117, 0x00117, NO_DUMP )
-	ROM_LOAD( "gal16v8b.u42",  0x0022e, 0x00117, NO_DUMP )
-	ROM_LOAD( "gal16v8b.u9",   0x00345, 0x00117, NO_DUMP )
+	ROM_LOAD( "gal16v8b.u41",  0x00000, 0x00117, CRC(fdb4cd61) SHA1(f510077b707864b2536942db6157118ca15922de) )
+	ROM_LOAD( "gal16v8b.s63",  0x00117, 0x00117, CRC(f4f78925) SHA1(738281c05c0f51c0edcf65bacc18ebc87c2408c1) )
+	ROM_LOAD( "gal16v8b.u42",  0x0022e, 0x00117, CRC(0cf55cec) SHA1(845395ca0587627331b9ac48777f7cb6b54b9401) )
+	ROM_LOAD( "gal16v8b.u9",   0x00345, 0x00117, CRC(8272668f) SHA1(9037f0d9c7625d05d2087e6f2d159dece934a945) )
```

Comment:
How confident are you with the dumps?  Were the devices unprotected?  These GALs can have registered outputs.

Reply:
All of them were unprotected. We desoldered from the PCB and dumped them with a Beeprog2, getting the same content as the other sets... 100% they're good dumps!

Reply:
That’s unusually good news :)